Scripts/Modules/SerializedPreferencesModule.cs

```diff
@@ -94,8 +95,10 @@ internal string SerializePreferences()
 				}
 
 				if (item != null)
-					preferences.items.Add(item);
+					payloads[serializer.GetType()] = item;
```

Comment:
Discussion result: Taking this a step further actually.  Storing the prefs in a dictionary and converting to list to serialize/deserialize.  Also spitting out warning if double-serialization tries to happen.